omero_search_engine/api/v1/resources/schemas/filter_schema.json

```diff
@@ -13,12 +13,12 @@
     },
      "value": {
       "name":"value",
-      "type": "string"
+      "type": ["array", "string"]
     },
      "operator": {
       "name": "operator",
       "type": "string",
-      "enum": ["equals", "not_equals", "contains","not_contains"]
+      "enum": ["equals", "not_equals", "contains","not_contains","in"]
```

Comment:
to match the existing logic. I assume a "not_in" will be introduced in a follow-up PR

Reply:
Yes, I think it is a good idea